be/src/common/config.h

```diff
@@ -1369,6 +1369,13 @@ CONF_String(rocksdb_cf_options_string, "block_based_table_factory={block_cache={
 
 CONF_String(rocksdb_db_options_string, "create_if_missing=true;create_missing_column_families=true");
 
+// It is the memory percent of write buffer for meta in rocksdb.
+// default is 5% of system memory.
+CONF_mInt64(rocksdb_write_buffer_memory_percent, "5");
```

Comment:
if the parameter won't be take effect in runtime, better define it as immutable, so it is clear that the parameter can only take effect after restart.

Reply:
shall also explain that the minimal size is 64MB even if `total_mem * percent < 64MB.`

Reply:
done